deploy-manage/users-roles/cluster-or-deployment-auth.md

```diff
@@ -1,16 +1,55 @@
 
-% GitHub issue: https://github.com/elastic/docs-projects/issues/347
+In this section, you’ll learn how to set up authentication and authorization at the cluster or deployment level, and learn about the underlying security technologies that Elasticsearch uses to authenticate and authorize requests internally and across services.
 
-% Use migrated content from existing pages that map to this page:
+This section only covers direct access to and communications with an Elasticsearch cluster - sometimes known as a deployment. To learn about managing access to your {{ecloud}} organization or {{ece}} orchestrator, or to learn how to use single sign-on to access a cluster using your {{ecloud}} credentials, refer to [Manage users and roles](/deploy-manage/users-roles.md).
 
-% - [ ] ./raw-migrated-files/cloud/cloud-enterprise/ece-securing-clusters.md
-% - [ ] ./raw-migrated-files/cloud/cloud/ec-security.md
\ No newline at end of file
+## Quickstart
+
+If you plan to use native Elasticsearch user and role management, then [follow our quickstart](/deploy-manage/users-roles/cluster-or-deployment-auth/quickstart.md) to learn how to set up basic authentication and authorization features.
+
+### User authentication
+
+Set up methods to identify users to the Elasticsearch cluster.
+
+Key tasks for managing user authentication include:
+
+* [Managing default users](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-users.md)
+* [Managing users natively](/deploy-manage/users-roles/cluster-or-deployment-auth/native.md)
+* [Integrating with external authentication providers](/deploy-manage/users-roles/cluster-or-deployment-auth/external-authentication.md)
+
+You can also learn the basics of Elasticsearch authentication, learn about accounts used to communicate within an Elasticsearch cluster and across services, and perform advanced tasks.
+
+[View all user authentication docs](/deploy-manage/users-roles/cluster-or-deployment-auth/user-authentication.md)
+
+### User authorization
+
+After a user is authenticated, use role-based access control to determine whether the user behind an incoming request is allowed to execute the request.
+
+Key tasks for managing user authorization include: 
+
+* Assigning [built-in roles](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-roles.md) or [defining your own](/deploy-manage/users-roles/cluster-or-deployment-auth/defining-roles.md)
+* [Mapping users and groups to roles](/deploy-manage/users-roles/cluster-or-deployment-auth/mapping-users-groups-to-roles.md)
+* [Setting up field- and document-level security](/deploy-manage/users-roles/cluster-or-deployment-auth/controlling-access-at-document-field-level.md)
+
+You can also learn the basics of Elasticsearch authorization, and perform advanced tasks.
+
+::::{tip}
+User roles are also used to control access to [spaces](/deploy-manage/manage-spaces.md).
```

Comment:
Should we say Kibana spaces instead of just spaces?
They are called simply spaces, but now that we are merging books I'm not sure if we should talk about Kibana spaces or just spaces.

Anyway this should be aligned with whatever way `manage-spaces.md` present the spaces, and that's not created yet :)

```suggestion
User roles are also used to control access to [{{kib}} spaces](/deploy-manage/manage-spaces.md).
```